app/gui/src/controller/visualization/manager.rs

```diff
@@ -285,11 +286,15 @@ impl Manager {
             // Early return: requested to remove visualization that was already removed.
             return;
         };
+        let prj = self.executed_graph.module_qualified_name(&*self.executed_graph.project);
+        let graph = self.executed_graph.graph();
+        let module = format!("{prj}.{}", graph.module.name());
```

Comment:
I believe this should be changed to

```rs
let graph = self.executed_graph.graph();
let module = graph.qualified_name();
```
The graph now contains the `project_name`, but its `qualified_name` method doesn't make use of it yet, so it has to be adjusted as well.

```rs
    // gui/src/controller/module.rs impl Handle

    /// Get the module's qualified name.
    pub fn qualified_name(&self) -> QualifiedName {
        QualifiedName::new_module(self.project_name.clone(), self.model.id())
    }
```

Reply:
Changed to:
```
        let prj = self.executed_graph.module_qualified_name();
        let graph = self.executed_graph.graph();
        let module = prj.new_child(graph.module.name());
```